### PR TITLE
Added login screen(with all the resolved changes)

### LINKED ISSUE #44: Login Screen
Create:
- [x] Sign In screen
- [ ] Sign Up screen: @Neha62-lit

### PR DESCRIPTION
**Pull request by Samdid**
### Related Issue
- Fixes # 44 .

### Checklist
- [x] Tested
- [x] No Conflicts
- [x] Change In Code
- [ ] Change In Documentation

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -56,7 +56,8 @@ flutter:
   assets:
     - assets/
     - assets/items/
-    - images/
+    - assets/items/google.png
```
Comment (himanshusharma89) [inline pubspec.yaml:59]: No changes in this file is required, line 58 will do the work

File: lib/widgets/splash.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import 'dart:async';
```
Comment (himanshusharma89) [inline lib/widgets/splash.dart:1]: This whole file is deleted now, not required.

File: lib/main.dart [hunk 3]
```diff
@@ -51,9 +55,9 @@ class MyApp extends StatelessWidget {
             textTheme:
                 GoogleFonts.poppinsTextTheme(Theme.of(context).textTheme)),
         routes: <String, WidgetBuilder>{
-          '/dashboard': (BuildContext context) => Dashboard(),
+          '/login': (BuildContext context) => LoginScreen(),
         },
-        home: Dashboard(),
+        home: Splash(),
```
Comment (himanshusharma89) [inline lib/main.dart:60]: The Splash screen is removed.

**Comment by himanshusharma89** [comment]
@Samdid You should have pulled the latest code before creating the PR. Do the following changes:

**Review by himanshusharma89 — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (Samdid) [reply pubspec.yaml:59]: Ok

File: lib/widgets/splash.dart (hunk 2 above)
Comment (Samdid) [reply lib/widgets/splash.dart:1]: Ok

File: lib/main.dart (hunk 3 above)
Comment (Samdid) [reply lib/main.dart:60]: Ok

**Comment by Samdid** [comment]
@himanshusharma89 All the changes are resolved

**Pull request merged**